charts/all/intel-device-plugins/templates/idp-sgxdeviceplugin.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+---
+apiVersion: deviceplugin.intel.com/v1
+kind: SgxDevicePlugin
+metadata:
+  name: sgxdeviceplugin-intel
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+spec:
+  image: registry.connect.redhat.com/intel/intel-sgx-plugin@sha256:1b89401a69bf664473b96fced1ded89551397b3b66e37a7a1b92d33bd3baccad
```

Comment:
Would it make sense to parameterize these values? Or would only one sha256 be valid at a time?

Reply:
I took this value from repo which is published by one of the Intel groups which helps with integration with OpenShift: https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/device_plugins/sgx_device_plugin.yaml

Reply:
Conventionally, values-global is for parameters that need to be identical in multiple charts. Since these values are only required in one chart apiece, it would be more appropriate to make them default parameters in their own charts' values.yaml files.

Reply:
As advised, Image names were moved to charts' values.yaml files. They are named in a way, that they could be overwritten from values-global if needed, in the future. I hope that this is what you were looking for.